torch/functional.py

```diff
@@ -1299,6 +1303,23 @@ def norm(input, p="fro", dim=None, keepdim=False, out=None, dtype=None):  # noqa
             else:
                 return _VF.norm(input, p, _dim, keepdim=keepdim, dtype=dtype, out=out)  # type: ignore
 
+    if isinstance(p, float):
```

Comment:
This is a pretty invasive change on our core codebase that makes this less readable - we should sync on other solutions before landing.